receiver/elasticapmreceiver/receiver.go

```diff
@@ -248,7 +249,18 @@ func (r *elasticAPMReceiver) processBatch(ctx context.Context, batch *modelpb.Ba
 				}
 			}
 		case modelpb.ErrorEventType:
-			// TODO
+			rl := ld.ResourceLogs().AppendEmpty()
+			gk := modelprocessor.SetGroupingKey{
+				NewHash: func() hash.Hash {
+					return xxhash.New()
+				},
+			}
+			if err := gk.ProcessBatch(ctx, batch); err == nil {
```

Comment:
Could there be more than one error even type in a batch? Looks like we will be processing the batch multiple times. Maybe adding a `processedBatch` guard? Or exposing the [processError method](https://github.com/elastic/apm-data/blob/main/model/modelprocessor/groupingkey.go#L45)}

Reply:
Ah, this is a good catch, this should not be in the for loop.

I moved this out before the loop - I think with that we are good.

What this does is that this generates the grouping key on errors - if this fails, the result is that there won't be a grouping key. But everything else would work. I think current check with printing an error is fine - this will cause issues by Kibana not being able to group errors on the APM UI - on the other hand, everything else will be still visible on the document in es.